defaults/main.yml

```diff
@@ -33,3 +33,17 @@ riemann_slack_tag: slack_alert
 
 # Disable verbose logging, useful for standing up the server.
 riemann_debug_logging: false
+
+riemann_create_folders:
+  - "conf.d"
+  - "utils"
```

Comment:
Alright, I can see the value here. We can always circle back to clean up, but I expect we'll ride out the separate dirs for the near future.